cpp/src/messaging_adapter.cpp

```diff
@@ -274,13 +277,6 @@ void on_link_local_open(messaging_handler& handler, pn_event_t* event) {
 
 void on_link_remote_open(messaging_handler& handler, pn_event_t* event) {
     auto lnk = pn_event_link(event);
-    // Currently don't implement (transaction) coordinator
-    if (pn_terminus_get_type(pn_link_remote_target(lnk))==PN_COORDINATOR) {
-      auto error = pn_link_condition(lnk);
-      pn_condition_set_name(error, "amqp:not-implemented");
-      pn_link_close(lnk);
-      return;
-    }
```

Comment:
I think this deletion isn't exactly correct: Whilst we do now need to cope with a remote coordinator this is only on the link the implementation itself has opened to control transactions not any other general link so we should still not allow incoming coordinator links in general.